modules/Bio/EnsEMBL/Compara/PipeConfig/LoadMembers_conf.pm

```diff
@@ -252,7 +252,7 @@ sub pipeline_analyses {
 # ---------------------------------------------[filter genome_db entries into reusable and non-reusable ones]------------------------
 
         {   -logic_name => 'check_reusability',
-            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::ProteinTrees::CheckGenomedbReusability',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::ProteinTrees::CheckGenomeReusability',
```

Comment:
```suggestion
            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::CheckGenomeReusability',
```

Reply:
Wrong module indeed, but it's `Bio::EnsEMBL::Compara::RunnableDB::ProteinTrees::CheckMembersReusability` that should be used